README.md

```diff
@@ -1,8 +1,21 @@
 # Knot.x Fragments Chrome Extension
-Extends the Developer Tools, adding a sidebar that displays Fragments data associated with the
-selected DOM element.
+Extends the Developer Tools, adding a sidebar that displays [Fragments](https://github.com/Knotx/knotx-fragments) 
+data associated with the selected DOM element.
 
-// TODO why do we need this?
+<p align="center">
+  <img src="assets/images/preview.gif" alt="Knot.x Fragments Chrome Extension"/>
```

Comment:
The gif is cool but if you were to rerecord it then take this into consideration:
- when fragment timeline items are clicked it isn't really clear what is going on (apart from them turning yellow). I think this can be skipped
- the legend can probably be skipped too, it just distracts a viewer from the main point 
- when in the graph view it would be good to zoom in on the graph, at least on the smaller one